Formula/fceux.rb

```diff
@@ -46,6 +33,6 @@ def install
   end
 
   test do
-    system "#{bin}/fceux", "-h"
+    system "#{bin}/fceux", "--help"
```

Comment:
Do you mind proposing a more substantial test?

>We want tests that don’t require any user input and test the basic functionality of the application. For example foo build-foo input.foo is a good test and (despite their widespread use) foo --version and foo --help are bad tests. 

https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
This is a Nintendo emulator. A more involved test would require loading a ROM (game). For legal reasons, no games are packaged with the emulator.  Users must go get the games themselves. Running the help is all the previous formula used to do and I have no interest in creating a more detailed test.

Reply:
How about trying to make it run a non existing ROM? `--help` tests rarely ever catch real issues so we'd rather check if the software works with a real test.

Reply:
An alternative that works for others is to briefly run the program and then check whether the files it creates while running exist. For examples, see `cdogs-sdl` or `cataclysm`.

Reply:
Opening a non-existent ROM is not a fatal error. It will just startup the GUI and show an error message that will then need to be acknowledged. Then the user can use the main menu to open a file browser and open a ROM that way. I want to avoid the program from hanging waiting for a user input when running the test.